CONTRIBUTING.md

```diff
@@ -0,0 +1,44 @@
+# Contributing to LMHT
+
+Thank you for your interest in contributing to LMHT! Enclosed are guidelines for helping us improve the source code and making LMHT an excellent learning tool.
+
+A couple of ways to contribute to LMHT is by submitting an issue or a pull request, or both.
+
+##Submitting an Issue
+
+Create a new issue at the [Issues page], if you notice any errors within the site or in the documentation.  We love hearing your feedback on any new features you would like to see, too.  
+
+A few guidelines for submitting an issue:
+
+1. Make sure the new issue you are submitting does not already exist as an issue.
+2. Consider how your request aligns with current structure and design of LMHT.
+3. Be very detailed in the description of your issue - screenshots are helpful.
+4. Let us know if you would like to make a pull request regarding the issue you are creating.
+
+## Become a GitHub Contributor
+
+As a contributor, the expectation is that you can use Git and GitHub comfortably enough to fork branches, make pull requests, and practice good commit hygiene.
+
+(These directions are straight from [Contributing to Open Source])
+
+1.  Fork the repository and clone it locally. Attach your local to the upstream repository by adding it as a remote. Pull the changes from upstream so that you are modifying current data when you submit a pull request. More in-depth instructions [here].
+2.  Create a branch for your edits. Depending on what you have loaded in your editor, `git checkout -b MyBranch` should suffice.
+3.  It is common practice to name your branch according to what it is fixing or updating. Before submitting these changes, verify that they work.
+4.  Because the project uses Ajax, you must run a local web server to see your changes.
+
+```
+$  cd myrepo
+$  python -m SimpleHTTPServer
+*  Running on [http://0.0.0.0:8000](http://0.0.0.0:8000/)
```

Comment:
This is in a code block so the typical markdown pieces don't work. (tldr: this doesn't make a link)